tslib.d.ts

```diff
@@ -32,3 +32,5 @@ export declare function __asyncValues(o: any): any;
 export declare function __makeTemplateObject(cooked: string[], raw: string[]): TemplateStringsArray;
 export declare function __importStar<T>(mod: T): T;
 export declare function __importDefault<T>(mod: T): T | { default: T };
+export declare function __classPrivateFieldGet(receiver: object, privateMap: WeakMap<object, any>): any;
+export declare function __classPrivateFieldSet(receiver: object, privateMap: WeakMap<object, any>, value: any): any;

```

Comment:
probably not, since the only existing things that use generic are the ones related to the module system.

suggestion: s/object/any to match the others

Reply:
@mheiber Updated to `any`. I left (and updated) the comment with the question about genericizing, though.